### PR TITLE
chore: Rebuild all models with kmc-17.0.329 to address missing low-frequency words

### LINKED ISSUE #246: chore: rebuild all lexical models to address missing low-frequency prefix words
**Note**: as this is a compiler bug, all models will likely need recompilation so that they may receive the fix's effects.

_Originally posted by @jahorton in https://github.com/keymanapp/keyman/issues/11074_

After 17.0 is released to stable, and the toolchain is deployed to the lexical-models repository, we need to increment the version number on all lexical models in order to get a fixed version of the models to end users.

### PR DESCRIPTION
**Pull request by darcywong00**
Fixes #246

* ~Update kmc to latest 17.0.329 stable~ That was split out to #277
* Rebuilds all lexical-models to address missing low-frequency words

Also notes HISTORY.md file of sil.km.ggc for #265

### CODE REVIEW

File: experimental/ptg/ptg.nan-latn-tw.taigipoj/HISTORY.md
Comment (mcdurdin) [inline experimental/ptg/ptg.nan-latn-tw.taigipoj/HISTORY.md:6]: I think this needs to say _why_ we are rebuilding with the updated compiler.

**Review by mcdurdin — CHANGES_REQUESTED** [review]

**Comment by DavidLRowe** [comment]
Waiting for @mcdurdin to review and give approval.

**Comment by mcdurdin** [comment]
Can we split the compiler version update into its own PR please? Then this becomes just maintenance, and issues such as the sil.lzz.laz update happening simultaneously to this won't have an impact.

File: package-lock.json
Comment (mcdurdin) [inline package-lock.json:1]: This file shouldn't be in this PR

**Review by mcdurdin — APPROVED** [review]
Approved with removal of package-lock.json changes

File: package-lock.json
Comment (darcywong00) [reply package-lock.json:1]: argh - I reverted package-lock.json but pushed to my fork instead of here. Will revert on a follow-on

Comment (mcdurdin) [reply package-lock.json:1]: Don't worry about it. It doesn't do any harm as it stands.

**Pull request merged**
